### PR TITLE
AB#6377 Fixed menu content padding

### PR DESCRIPTION
**Pull request by adoprog**

### CODE REVIEW

File: Website/src/rendering/src/assets/css/layout/hero.css [hunk 1]
```diff
@@ -1,7 +1,5 @@
 .section__hero {
   @apply m-0 bg-black bg-left bg-no-repeat bg-cover relative;
-
-  margin-top: 122px;
```
Comment (jflheureux) [inline Website/src/rendering/src/assets/css/layout/hero.css:4]: At the bottom of this file, there is an alteration for Experience Editor:

```css
.experience-editor-active {
  .section__hero {
    @apply mt-0;
  }
}
```

This likely has to be moved and modified to affect the `<main>` element when in the Experience editor with the top menu that is not sticky.

There is logic in `Layout.tsx` regarding this EE support as well.

File: Website/src/rendering/src/assets/css/base/sitecore.css [hunk 2]
```diff
@@ -12,4 +12,12 @@
   .scpm {
     display: none !important;
   }
+
+  main {
```
Comment (jflheureux) [inline Website/src/rendering/src/assets/css/base/sitecore.css:16]: This file is intended for CSS styles in relation to Sitecore tools like Horizon and Experience Editor. Could we move the "main" properties to another file. I have not found a proper file to put it into so we could either create a main.css or layout.css file in the "layout" folder for the main page layout styles.

**Review by jflheureux — CHANGES_REQUESTED** [review]

File: Website/src/rendering/src/assets/css/base/sitecore.css (hunk 2 above)
Comment (adoprog) [reply Website/src/rendering/src/assets/css/base/sitecore.css]: Done.

File: Website/src/rendering/src/assets/css/layout/hero.css (hunk 1 above)
Comment (adoprog) [reply Website/src/rendering/src/assets/css/layout/hero.css:4]: Done.

File: Website/src/rendering/src/assets/css/base/sitecore.css [hunk 3]
```diff
@@ -12,4 +12,8 @@
   .scpm {
     display: none !important;
   }
+
+  .experience-editor-active {
```
Comment (jflheureux) [inline Website/src/rendering/src/assets/css/base/sitecore.css:16]: For some reason, in my portal instance, this directive is not bundled in the final CSS file and the EE shows a white 122px gap between the header and the main section.
https://jel-pull0076-cm.sitecoredemo.com/?sc_mode=edit
I will continue to troubleshoot in a couple of hours.

Comment (jflheureux) [reply Website/src/rendering/src/assets/css/base/sitecore.css:16]: This was caused by Tailwindcss purging unused definitions in production builds. We were missing some source files in the purge configuration.

**Review by jflheureux — APPROVED** [review]

**Pull request merged**